implemented initial gradle build file structure

jstrelka opened the pull request:
The Gradlers team has implemented a successful HelloWorld.java build using Gradle to address User Story #60

jstrelka commented:
Closed due to HelloWorld.java missing will reopen new request with required files

jody commented:
Please provide information about the additions/modifications in a form appropriate to _current_ CS3250 class members; for example, the intended function and where to find relevant information.

Why does DevelopmentConventions.md need to be changed?

Please ensure adherence to Java coding conventions (package naming, etc.).

Consider updating boiler-plate documentation.

jstrelka commented:
Dr. Paul I believe I have fixed all of the changes you requested regarding the DevelopmentConventions.md file and the .gitignore issue. 

Gradle automatically wipes the .gitignore file when initialized for the first time and writes its own structure.

The DevelopmentConventions.md file was not tracked correctly initially because I had not realized my windows linux subsystem terminal was not restarted the first time I removed the project from my local device and re-cloned it using my windows command prompt. 

All changes now display in diff as expected, sorry for the confusion.
